wnfs-wasm/src/fs/private/forest.rs

```diff
@@ -50,107 +31,37 @@ impl PrivateForest {
         Self(Rc::new(WnfsPrivateForest::default()))
     }
 
-    #[wasm_bindgen]
-    pub fn get(&self, private_ref: PrivateRef, store: BlockStore) -> JsResult<Promise> {
+    /// Loads an existing private forest from a given CID
+    /// You need to have previously `.store()`ed it to get its CID.
+    pub fn load(cid: Vec<u8>, store: BlockStore) -> JsResult<Promise> {
```

Comment:
This diff is horrible :sweat_smile: 

I've removed all previous functions from `PrivateForest`'s Wasm API (`put` and `get`).

Instead, I realized we still needed `store` and `load` for webnative, so these got added.